src/client/components/Options/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+import { ChoiceContext, DelayContext } from 'contexts/Home';
+import 'styles/options.css';
+
+const Options = () => {
+  const { choice, setChoice } = useContext(ChoiceContext);
+  const { delay, setDelay } = useContext(DelayContext);
+
+  const handleChoiceChange = (event) => {
+    setChoice(event.target.value);
+  };
+  
+  const handleDelayChange = (event) => {
+    setDelay(event.target.value);
+  };
+
+  return (
+    <section className="options">
+      <FormControl component="fieldset" className="options">
+        <FormLabel component="legend">Choice</FormLabel>
+        <RadioGroup row aria-label="position" name="position" value={choice} onChange={handleChoiceChange}>
+          <FormControlLabel
+            value="1"
+            control={<Radio color="primary" />}
+            label="Cats"
+          />
+          <FormControlLabel
+            value="2"
+            control={<Radio color="primary" />}
+            label="Sharks"
+          />
+          <FormControlLabel
+            value="3"
+            control={<Radio color="primary" />}
+            label="Mix"
+          />
+        </RadioGroup>
```

Comment:
Instead of radio buttons, can you use 2 buttons? 
<img width="213" alt="Screen Shot 2021-05-31 at 5 13 40 PM" src="https://user-images.githubusercontent.com/35695190/120242733-9f7fb700-c233-11eb-90ef-51ab026e9148.png">
The user can either click on one or both depending on what animals they would like to see